src/Generating/CompoGenBiomal.cpp

```diff
@@ -33,7 +33,7 @@ class cPattern
 
 	constexpr cPattern(std::initializer_list<BlockInfo> a_TopBlocks)
```

Comment:
You need to adjust the class' doxycomment, too, the `256` is a `cChunkDef::Height` there.